fixtures/initial_data.json

```diff
@@ -0,0 +1,215 @@
+      "is_active": true,
+      "date_joined": "2017-06-29T01:53:31",
+      "groups": [],
+      "user_permissions": [
+        55,
+        53,
+        67
+      ]
+    }
+  },
+  {
+    "model": "auth.user",
+    "pk": 4,
+    "fields": {
+      "password": "pbkdf2_sha256$30000$aAAPSeaUZ4EB$W16ENSbjxBRhiEnDIwk3zBx8YXwvDlbT+YuqZ0EqM1U=",
+      "last_login": "2017-06-29T22:54:18",
+      "is_superuser": false,
+      "username": "pepe",
+      "first_name": "",
+      "last_name": "",
+      "email": "pepe@hackupc.com",
+      "is_staff": false,
+      "is_active": true,
+      "date_joined": "2017-06-29T01:54:45",
+      "groups": [],
+      "user_permissions": []
+    }
+  },
+  {
+    "model": "account.emailaddress",
+    "pk": 2,
+    "fields": {
+      "user": 2,
+      "email": "devs@hackupc.com",
+      "verified": true,
+      "primary": true
+    }
+  },
+  {
+    "model": "account.emailaddress",
+    "pk": 3,
+    "fields": {
+      "user": 3,
+      "email": "jon@hackupc.com",
+      "verified": true,
+      "primary": true
+    }
+  },
+  {
+    "model": "account.emailaddress",
+    "pk": 4,
+    "fields": {
+      "user": 4,
+      "email": "pepe@hackupc.com",
+      "verified": true,
+      "primary": true
+    }
+  },
+  {
+    "model": "register.hacker",
+    "pk": "2ceb6d17-dadd-4f47-b3e7-f29d69eb1d7b",
+    "fields": {
+      "user": 3,
+      "name": "Jon",
+      "lastname": "Snow",
+      "gender": "M",
+      "graduation_year": 2017,
+      "university": "THE NORTH",
+      "degree": "Nightwatc",
+      "github": "",
+      "devpost": "",
+      "linkedin": "",
+      "site": "https://www.youtube.com/watch?v=dQw4w9WgXcQ",
+      "diet": "None",
+      "tshirt_size": "L"
+    }
+  },
+  {
+    "model": "register.hacker",
+    "pk": "380f8d87-403e-41b3-9b7e-c01ae31446ab",
+    "fields": {
+      "user": 4,
+      "name": "Pepe",
+      "lastname": "Biene",
+      "gender": "M",
+      "graduation_year": 2017,
+      "university": "Universitat Politecnica de Catalunya",
+      "degree": "Computer Science",
+      "github": "",
+      "devpost": "",
+      "linkedin": "",
+      "site": "https://www.youtube.com/watch?v=dQw4w9WgXcQ",
+      "diet": "None",
+      "tshirt_size": "M"
+    }
+  },
+  {
+    "model": "register.application",
+    "pk": "078e119f0783a995ec1ad3c714bf44af",
+    "fields": {
+      "hacker": "2ceb6d17-dadd-4f47-b3e7-f29d69eb1d7b",
+      "edition": "F17",
+      "submission_date": "2017-06-29T01:57:56",
+      "status_update_date": "2017-06-30T03:09:30",
+      "sendgrid_id": "",
+      "under_age": false,
+      "first_timer": true,
+      "description": "ALL OF IT",
+      "projects": "I WRITE CODE",
+      "scholarship": false,
+      "origin_city": "Boulder",
+      "origin_country": "United States of America",
+      "lennyface": "-.-",
+      "resume": "https://www.youtube.com/watch?v=dQw4w9WgXcQ",
+      "team": false,
+      "teammates": "",
+      "authorized_mlh": true,
+      "status": "P",
```

Comment:
Ah, didn't mention it in my commit but I changed this one to P. So that there's at least one application visible in the vote interface after applying the fixture.